features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -95,6 +98,77 @@ export default function ListBlacklistThrottlingPolicies() {
         }),
     };
 
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
+        let incrementId = 0;
+        return new Promise(((resolve, reject) => {
```

Comment:
Do we need to create a new Promise here?, Is this because of that Alert message issue ?

Reply:
Nope not for the alert.
We need to send a new promise for it to get resolved in the [ListBase](https://github.com/wso2/carbon-apimgt/blob/admin_portal_revamp/features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/AdminPages/Addons/ListBase.jsx#L87)

Reply:
It's not good to add additional async operations unless it's absolutely needed, I don't see any requirement to create a promise object here ?

Reply:
Fixed with commit `a0b7e46`